calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx

```diff
@@ -0,0 +1,94 @@
+---
+description: Get alerts on threats in a single dashboard.
+---
+
+# Security event management
+
+:::note
+
+This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
+
+:::
+
+## Big picture
+
+Get alerts on security events in a single dashboard.
+
+## Value
+
+Security events indicate that a threat actor may be present in your Kubernetes cluster. For example, a DNS request to a malicious hostname, a triggered WAF rule, or the opening of a sensitive file. {{prodname}} provides security engineers and incident response teams with a single dashboard to manage threat alerts. Benefits include:
+
+- A filtered list of critical events with recommended remediation
+- Identify impacts on applications
+- Understand the scope and frequency of the issue
+- Manage alert noise by dismissing events (show/hide)
+
+## Before you begin
+
+**Required**
+
+- [Container threat detection is enabled](./container-threat-detection)
+
+**Limitations**
+
+- You cannot control which users can view or edit the page using fine-grained role-based access controls
+
+## Security events dashboard
+
+In Manager UI, go to **Threat defense**, **Security Events**.
+
+### UI help
+
+**Event details page**
+
+Provides actions to remediate the detection and stop the attack from progressing. For example:
+
+![runtime-security](/img/calico-cloud/runtime-security-events.png)
+
+**Severity**
+
+{{prodname}} calculates severity (Critical, High, Medium, Low) using a combination of NIST CVSS 3.0 and MITRE IDs.
```

Comment:
Not sure how accurate that statement is. We (security researcher) provide a severity score that we then map a level inspired by the thresholds of NIST CVSS 3.0. I don’t think Mitre IDs have a direct input in the severity score.

When I read it I get the impression that Mitre IDs contain some kind of severity information. My understanding is that it’s not the case and this is a judgement call from our security researcher. @anthonytwh do you have any thoughts on that?

Reply:
Yeah MITRE Tactic or CVSS are not used for the severity of a detector. It is an arbitrary number we assign at the moment (no caluclations). 

I think we can simplify this wording somewhat along the lines of: a severity of an alert is a score given to a detection based upon the risk assessment of the alert.

Reply:
I changed this to: Alert severity is a score given to a detection based upon the risk assessment of the alert.